lib/builder/webpack/utils/postcss.js

```diff
@@ -115,11 +121,14 @@ export default class PostcssConfig {
         this.preset = config.preset
         delete config.preset
       }
-      _.defaults(config, this.defaultConfig)
-
-      this.loadPlugins(config)
+      if (Array.isArray(config.plugins)) {
```

Comment:
When plugins can/should be array?

Reply:
If plugins is array,  will just treat it as option plugins of postcss-loader and override all default plugins.
As it can be array before, so this part is for back compatibility.

Reply:
Ah, i see, i got that it will override if its array, i was wondering why it can be array :) Didnt saw it as array before. But make sense.